arrocke/gloss-translation
# chore: import unabridged BDB

## What has changed

* Render lexicon entries as sanitized HTML rather than Markdown. The BDB has super and subscripts which don't have Markdown syntax. And we need to be able to mark text as RTL within a sentence.
* Update the LSJ import to convert to HTML
* Update the BDB import to use the unabridged BDB aligned to strongs numbers
* Wrap greek and hebrew in lexicon entires in spans with classes to be able to set the appropriate font
* Remove the strongs concordance import
* Expanded the sidebar to make it more readable on wider screens



## Connected Issues

closes #280 



## QA Steps



1. Run `DELETE FROM "LemmaResource"` in the database
2. Run `nx run db:import-bdb` and `nx run db:import-lsj`
3. Load a verse from the OT and NT
4. Check lexicon entries for formatting and fonts

## Post-Deployment



I'll replace the LemmaResources table with the correct data
